pom.xml

```diff
@@ -430,7 +430,7 @@ under the License.
 
     <camel.version>3.13.0</camel.version>
 
-    <slf4j.version>2.0.0-alpha5</slf4j.version>
+    <slf4j.version>1.7.32</slf4j.version>
```

Comment:
Are we switching back here to 1.7.32 in order to align with CAS, correct?

Reply:
Yes. Staying on alpha seems to cause a few issues:

1. Seems like Boot 2.6 RC1 has issues recognizing this given logger binding have changed; possibly because boot's dependency management of slf4j has changed.
2. The alpha version is quite unstable, does not work against graalvm and native-images with the spring-native project. This is the main reason for CAS to fallback on 1.7.x.

If preferable, I can try to only override this for CAS. I see that slf4j has been on alpha for a long time, and there is no official/public plan as to when this might get released. Log4j is also waiting for slf4j to stabilize before the build compatible bindings for it. So what works now, may not work in the future, and is in fact sort of luckily works.

Reply:
Not a big deal, let's stick with `1.7.32` everywhere, thanks.